src/spork.cpp

```diff
@@ -84,7 +84,10 @@ bool CSporkManager::UpdateSpork(int nSporkID, int64_t nValue)
 {
     CSporkMessage spork = CSporkMessage(nSporkID, nValue, GetTime());
 
-    spork.Sign(strMasterPrivKey);
+    if (!spork.Sign(strMasterPrivKey)) {
```

Comment:
Transition to evo depends on some particular spork to be set at block nDIP0003EnforcementHeight. This change basically disables LLMQ.

The plan is to remove all the spork code after mainnet transitioned to evo we need it to stay here. Are you sure the ban is because of incorrect sporks?

Reply:
yes, it reaches this line (https://github.com/zcoinofficial/zcoin/blob/core14/src/spork.cpp#L53). My change just prevents messages which contain incorrect signatures to be broadcasted.

Reply:
I changed spork handling. LLMQ doesn't need spork anymore. And I changed `spork.cpp`. Please update your PR

Reply:
updated.